tests/data/parity/trace1.json

```diff
@@ -0,0 +1,1997 @@
+[
```

Comment:
Not going to start with `trace0.json`? 😄

Reply:
Would it make sense to name the JSON files by case name?

`error_trace.json`
`lots_of_precompiles.json`
`very_long.json`

That way, it will help keep track of what we are covering.